server/src/test/java/org/apache/kafka/server/metrics/ClientMetricsTestUtils.java

```diff
@@ -40,18 +40,18 @@ public class ClientMetricsTestUtils {
 
     public static final String DEFAULT_METRICS =
         "org.apache.kafka.client.producer.partition.queue.,org.apache.kafka.client.producer.partition.latency";
-    public static final int DEFAULT_PUSH_INTERVAL_MS = 30 * 1000; // 30 seconds
-    public static final List<String> DEFAULT_CLIENT_MATCH_PATTERNS = List.of(
+    public static final int DEFAULT_INTERVAL_MS = 30 * 1000; // 30 seconds
```

Comment:
Should we align the naming? `DEFAULT_INTERVAL_MS` -> `INTERVAL_MS_DEFAULT`

Reply:
I've changed the naming but because these are not the true default, but the defaults for the test, I used `_TEST_DEFAULT` in the names.